### PR TITLE
Development

### PR DESCRIPTION
**Pull request by EttoreFoti**
waiting for approval on ci steps and refactored tree

### CODE REVIEW

File: crossplane.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: meta.pkg.crossplane.io/v1alpha1
+kind: Configuration
+metadata:
+  name: krateo-template-aws-stack
+  annotations:
+    meta.crossplane.io/maintainer: Diego Braga <diego.braga@kiratech,it>
+    meta.crossplane.io/keywords: krateo, aws, template, kubernetes, platform, reference
+    meta.crossplane.io/source: github.com/krateoplatformops/krateo-module-core
```
Comment (braghettos) [inline crossplane.yaml:8]: Wrong source

File: crossplane.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    meta.crossplane.io/readme: |
+      This reference platform `Configuration` for Kubernetes and Data Services
+      is a starting point to build, run, and operate your own internal cloud
+      platform and offer a self-service console and API to your internal teams.
+
+      It provides platform APIs to provision fully configured Krateo Core
+spec:
+  crossplane:
+    version: ">=v1.5.0-0"
```
Comment (braghettos) [inline crossplane.yaml:23]: Update to version 1.7.0

File: crossplane.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  # dependsOn:
```
Comment (braghettos) [inline crossplane.yaml:24]: I would enable this section: which are the providers we need in this composition?

File: defaults/aws-stack-package.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: krateo-template-aws-stack
+spec:
+  package: ghcr.io/krateoplatformops/krateo-template-aws-stack:development
```
Comment (braghettos) [inline defaults/aws-stack-package.yaml:6]: Let's point to the latest

Comment (EttoreFoti) [reply defaults/aws-stack-package.yaml:6]: here I'm pointing to development to let test run smoothly without changing it manually, the same file in the main branch should point to latest

**Pull request merged**